admin/src/main/ts/app/users/details/user-details.component.ts

```diff
@@ -203,47 +205,47 @@ export class UserDetails implements OnInit, OnDestroy{
         private route: ActivatedRoute,
         private router: Router,
         private userListService: UserListService
-    ){}
+    ) {
+    }
 
     ngOnInit() {
         this.dataSubscriber = this.usersStore.onchange.subscribe(field => {
-            if(field === 'user') {
-                if(this.usersStore.user &&
-                        !this.usersStore.user.structures.find(
-                            s => this.usersStore.structure.id === s.id)) {
+            if (field === 'user') {
+                if (this.usersStore.user &&
+                    !this.usersStore.user.structures.find(
+                        s => this.usersStore.structure.id === s.id)) {
```

Comment:
j'ai l'habitude de le voir dans l'autre sens (s => s.id ===  this.usersStore.structure.id) est ce qu'il existe une convention ?

Reply:
pas à ma connaissance!